transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/internal/LogResponse.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.cct.internal;
+
+import android.util.JsonReader;
+import android.util.JsonToken;
+import androidx.annotation.Nullable;
+import com.google.auto.value.AutoValue;
+import java.io.IOException;
+import java.io.StringReader;
+
+@AutoValue
+public abstract class LogResponse {
+  /** Client should wait for next_request_wait_millis before sending the next log request. */
+  public abstract long getNextRequestAwaitMillis();
+
+  /**
+   * Quality of Service tiers enforced by server for overriding client qos_tier setting in event
+   * logging.
+   */
+  @Nullable
+  public abstract QosTiersOverride getQosTiersOverride();
```

Comment:
We don't need these, the only thing that we implement at this point is `next_request_wait_millis`, so it's worth removing the getter and the classes

Reply:
Done. PTAL